addons/crm/views/crm_lead_views.xml

```diff
@@ -535,7 +535,7 @@
                     <templates>
                         <t t-name="kanban-box">
                             <t t-set="lost_ribbon" t-value="!record.active.raw_value and record.probability and record.probability.raw_value == 0"/>
-                            <div t-attf-class="#{!selection_mode ? kanban_color(record.color.raw_value) : ''} #{lost_ribbon ? 'oe_kanban_card_lost_ribbon' : ''} oe_kanban_global_click oe_kanban_card d-flex flex-column">
+                            <div t-attf-class="#{!selection_mode ? kanban_color(record.color.raw_value) : ''} #{lost_ribbon ? 'oe_kanban_card_ribbon' : ''} oe_kanban_global_click oe_kanban_card d-flex flex-column">
```

Comment:
Mmmh ? Seems like unrelated change / fix ?

Reply:
I renamed this css class because it is used for lost ribbons as well as won ribbons (I added the won ribbon for forecast in this commit), and this class is used to add the ribbon, not the content of the ribbon, so I think this name makes more sense.

Reply:
Ok